tests/components/kaleidescape/test_media_player.py

```diff
@@ -37,9 +38,12 @@ async def test_entity(
 ) -> None:
     """Test entity attributes."""
     entity = hass.states.get(ENTITY_ID)
-    assert entity is not None
+    entry = er.async_get(hass).async_get(ENTITY_ID)
+    assert entity
     assert entity.state == STATE_OFF
     assert entity.attributes["friendly_name"] == FRIENDLY_NAME
+    assert entry
+    assert entry.unique_id == MOCK_SERIAL
```

Comment:
Added for constancy with sensor test.

Reply:
Usually we don't want to modify two platforms in the same PR.

https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=small&_highlight=po#5-make-your-pull-request-as-small-as-possible

Reply:
Removed.